src/TritonCTS/README.md

```diff
@@ -26,7 +26,7 @@ clock_tree_synthesis -buf_list "BUF_X1 BUF_X2" \
                      -wire_unit 20 \
                      -clk_nets "clk" \
                      -out_path "/home/myfolder/" \
-                     -only_characterization 0
+                     -characterization_only 1
```

Comment:
If it is a flag the 1 shouldn't be there.
But that contradicts the call on the left where it is disabled, so shouldn't the
flag not even be there?